examples/disk_pressure_read_safemode.yaml

```diff
@@ -0,0 +1,23 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+apiVersion: chaos.datadoghq.com/v1beta1
+kind: Disruption
+metadata:
+  name: disk-pressure-read
+  namespace: chaos-demo
+spec:
+  level: pod
+  selector:
+    app: demo-curl
+  count: 1
+  unsafeMode:
+    disableAll: false
```

Comment:
Do we need `DisableSpecificContainDisk` here?

Reply:
Yes, nice catch.

Reply:
yes, good catch